pkg/defaults/complianceoperator/benchmarks.go

```diff
@@ -0,0 +1,57 @@
+package complianceoperator
+
+import (
+	"embed"
+	"path/filepath"
+
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/errorhelpers"
+	"github.com/stackrox/rox/pkg/features"
+	"github.com/stackrox/rox/pkg/jsonutil"
+	"github.com/stackrox/rox/pkg/logging"
+	"github.com/stackrox/rox/pkg/utils"
+)
+
+const (
+	benchmarkDir = "files"
+)
+
+var (
+	log = logging.LoggerForModule()
+
+	//go:embed files/*.json
+	benchmarksFS embed.FS
+)
+
+func LoadComplianceOperatorBenchmarks() ([]*storage.ComplianceOperatorBenchmarkV2, error) {
+	if !features.ComplianceEnhancements.Enabled() {
+		return nil, nil
+	}
+	files, err := benchmarksFS.ReadDir(benchmarkDir)
+	utils.CrashOnError(err)
```

Comment:
**important**
Do not crash, same reason as above.

Reply:
Please use `ShouldErr` instead.
I agree that this is an important part of the feature lifecycle for benchmarks, for this please write a unit test to check that all provided benchmarks files are valid.
This would eliminate most of the reasons to crash at runtime.

Reply:
I've returned the error 👍